### DISCUSSION TITLE
What if I have too many tabs?

### DISCUSSION THREAD

**Discussion by arcseedai** [Comfy-Org/ComfyUI#6263] [Q&A] [answered]
I am doing a project were I send requests to my server based on a script that builds workflows, the issue is, I have a front end that  builds the workflows, then it is opened on the server.

Now that we have  multiple tabs, if I keep on doing this I will be adding more and more tabs to it and the count of tabs will be huge at the end of a week for instance, I believe that will hinders performance or something like that.

Is there a possibility to turn the tabs function off or find some other solution?

**Comment by LukeG89** [comment, marked as answer]
You can select `Sidebar` in "Opened workflows position" to make the workflows open inside the sidebar window:

![image](https://github.com/user-attachments/assets/09a1c051-3999-46a4-971b-9494da66e25f)

Or you can right-click on the workflow tab you want to keep open and select "Close Other Tabs"

![image](https://github.com/user-attachments/assets/ce5ad723-adfd-4886-814b-e1ef6acbb319)

**Reply by arcseedai** [reply]
Thank you very much, I will check that out!
